src/hourly-operating/hourly-operating.controller.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  ClassSerializerInterceptor,
+  Controller,
+  Get,
+  Query,
+  UseInterceptors,
+} from '@nestjs/common';
+import { ApiOkResponse, ApiSecurity, ApiTags } from '@nestjs/swagger';
+import { HourlyOperatingService } from './hourly-operating.service';
+import { HourlyOperatingParamsDto } from '../dto/hourly-operating.params.dto';
+import { HourlyOperatingDTO } from '../dto/hourly-operating.dto';
+
+@ApiTags('Emissions')
+@Controller('emissions')
+@ApiSecurity('APIKey')
+export class HourlyOperatingController {
+  constructor(private readonly service: HourlyOperatingService) {}
+
+  @Get('hourly-operating-data')
```

Comment:
Need to add this for the multi select parameters orisCode and locationName. Right now the transform in the params dto is causing an error when trying to pass multiple values on swagger because it can't split on | 
```
 @ApiQuery({
    style: 'pipeDelimited',
    name: 'orisCode',
    required: true,
    explode: false,
  })
 @ApiQuery({
    style: 'pipeDelimited',
    name: 'locationName',
    required: true,
    explode: false,
  })
```